### PR TITLE
new optimisation 

### PR DESCRIPTION
**Pull request by Blondi003**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,14 @@
-import React from 'react';
-import './App.css';
-import CounterExample from './CounterExample';
+import React, { useState } from 'react';
 
-function App() {
-  return <div>{<CounterExample />}</div>;
-}
+function CounterExample() {
```
Comment (koniuszy) [inline src/App.js:3]: you can use arrow function as well :), try to do it

File: src/App.js [hunk 2]
```diff
@@ -1,9 +1,14 @@
-import React from 'react';
-import './App.css';
-import CounterExample from './CounterExample';
+import React, { useState } from 'react';
 
-function App() {
-  return <div>{<CounterExample />}</div>;
-}
+const App = () => {
```
Comment (koniuszy) [inline src/App.js:3]: nice! it looks amazing,

you may try export default inline this component now 🥰

Comment (Blondi003) [reply src/App.js:3]: How to export this component?

Comment (koniuszy) [reply src/App.js:3]: inline -> that means in one line only, you can export default function without using `const`

Comment (koniuszy) [inline src/App.js:3]:
```suggestion
export default () => {
```

File: src/App.js [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
-export default App;
+  return (
+    <div>
+      <h1>{count}</h1>
+      <button onClick={() => setCount(count + 1)}>Plus</button>
+      <button onClick={() => setCount(count - 1)}>minus</button>
+    </div>
+  );
+};
+export default () => {};
```
Comment (koniuszy) [inline src/App.js:14]:
```suggestion
```

**Pull request merged**